table/block_based/block_based_table_reader.cc

```diff
@@ -3180,7 +3180,8 @@ Status BlockBasedTable::GetKVPairsFromDataBlocks(
 }
 
 Status BlockBasedTable::DumpTable(WritableFile* out_file) {
-  auto out_file_wrapper = WritableFileStringStreamAdapter(out_file);
```

Comment:
IIRC gcc 4.8 has an incomplete / non-final implementation of C++11. I suspect that this would work:
`WritableFileStringStreamAdapter out_file_wrapper(out_file);`

Reply:
You're right, that works. Thanks.